node/src/blockchain/blockchain_bridge.rs

```diff
@@ -601,6 +605,17 @@ mod tests {
         }
     }
 
+    impl SubsFactory<BlockchainBridge, BlockchainBridgeSubs>
+    for SubsFactoryTestAddrLeaker<BlockchainBridge>
```

Comment:
I would keep it in a `test_utils.rs` file.

Reply:
It implements closely on the BlockchainBridge. I think it belongs in that file only. This is not a piece of code of general use.